cpp/example_code/ses/tests/ses_gtests.cpp

```diff
@@ -67,7 +84,8 @@ void AwsDocTest::SES_GTests::AddCommandLineResponses(
 }
 
 bool AwsDocTest::SES_GTests::suppressStdOut() {
-    return std::getenv("EXAMPLE_TESTS_LOG_ON") == nullptr;
+   // return std::getenv("EXAMPLE_TESTS_LOG_ON") == nullptr;
+   return false; // Temporary override to debug testing framework.
```

Comment:
Why comment out instead of setting the environment correctly in weathertop?

Reply:
EXAMPLE_TESTS_LOG_ON enables capture of std::cout logging, which is usually stuff like "Bucket xxx was successfully created." Logging to std::cerr is always captured. By default I suppress std::cout, because it is not useful. It just creates a lot of noise. It could be useful, but not in this case,

I only wish to log for this file. And the current code is missing useful trace logging for mocks. I have to submit a PR anyway, so this is the best approach.

I will rollback these changes when I solve the problem.